src/HotChocolate/Skimmed/src/Skimmed/BuiltIns/SemanticNonNullDirectiveDefinition.cs

```diff
@@ -0,0 +1,19 @@
+using HotChocolate.Language;
+using DirectiveLocation = HotChocolate.Types.DirectiveLocation;
+
+namespace HotChocolate.Skimmed;
+
+public sealed class SemanticNonNullDirectiveDefinition : DirectiveDefinition
+{
+    internal SemanticNonNullDirectiveDefinition(ScalarTypeDefinition intType)
+        : base(BuiltIns.SemanticNonNull.Name)
+    {
+        IsSpecDirective = true;
```

Comment:
Is it?

Reply:
A spec directive does not need to be declared and must be defined by the core spec. I think this one will never be a spec directive but is just a bridge.

Reply:
It can still be a built-in directive.